# HG changeset patch
# User Jaroslav Hajek <highegg@gmail.com>
# Date 1264160631 -3600
#      Fri Jan 22 12:43:51 2010 +0100
# Branch release-3-2-x
# Node ID 19ab747d3c42256a0ed6e4310b60b43178876e33
# Parent  e320928eeb3aa2370b792e83dafc3e0ddecdc871
Added tag release-3-2-4 for changeset a34752a3aa8a

diff --git a/.hgtags b/.hgtags
--- a/.hgtags
+++ b/.hgtags
@@ -42,8 +42,9 @@ 739141cde75a22b9ac9b5b327ea30c61fd357748
 bd1b1fe9c6e952202bf5878cd042a168f6df6ddd ss-3-1-53
 3c3cbe8f18e03c73cc34494c8037d8a45a30bf7d ss-3-1-54
 5e276a0b999747b72af34e72eb63f64640da6399 ss-3-1-55
 a1d20052517a950033896f5a876de8f3ee520138 release-3-2-0
 a1d20052517a950033896f5a876de8f3ee520138 release-3-2-0
 530dc15eb653d538aaab0a139093354fbb6f0c68 release-3-2-0
 6607a20f0ab11a60026b8424df2da361ced53979 release-3-2-2
 ea2283b9436d606f71a2cf1cd3e0853f67741fc7 release-3-2-3
+a34752a3aa8a37a17dfb5d49de5c644ffb7713dc release-3-2-4
